core/indexer/workItems/workItemRemoveBlock.go

```diff
@@ -0,0 +1,49 @@
+package workItems
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+)
+
+type itemRemoveBlock struct {
+	indexer       removeIndexer
+	bodyHandler   data.BodyHandler
+	headerHandler data.HeaderHandler
+}
+
+// NewItemRemoveBlock will create a new instance of itemRemoveBlock
+func NewItemRemoveBlock(
+	indexer removeIndexer,
+	bodyHandler data.BodyHandler,
+	headerHandler data.HeaderHandler,
+) WorkItemHandler {
+	return &itemRemoveBlock{
+		indexer:       indexer,
+		bodyHandler:   bodyHandler,
+		headerHandler: headerHandler,
+	}
+}
+
+// Save wil remove block and miniblocks from elasticsearch database
+func (wirb *itemRemoveBlock) Save() error {
```

Comment:
This name of method is confusing because name is "Save" but method remove something from database. This method actually save an item for removing in elasticsearch database. To respect the interface I will have to keep this name.

Reply:
We might change this in the future. Save -> Execute